resources/constraints.yaml

```diff
@@ -27,3 +27,17 @@ test:
   max_runtime_seconds: 28800
   cores: 8
   min_vol_size_mb: 131072
+
+ssd1h8c:
```

Comment:
do you really want to put the volume type first as if it were the most important?
I'd rather stick with our current approach: {duration}{cores}{whatever comes next}

Reply:
I don't care either way, but `{whatever comes next}` was (as far as I was aware) never specified to be after the other fields (or before, for that matter). I went with a prefix because I figured it was less likely to lead to mistakes for me personally (I'm using to typing `1h8c`, I can't forget about `ssd` if I prefix it whereas muscle memory may lead me to skip it the other way around). Nevertheless I don't mind changing it.

Reply:
done